apps/accounts/models/test_models.py

```diff
@@ -46,21 +46,18 @@ class TestHostingProvider:
             "compensatie",
         ),
     )
+    @pytest.mark.django_db
```

Comment:
I see you using this decorator over paassing `db` into the function, and we have a mix of use throughout our test suite.

I'm open to us settling on either, but I'm not sure about the reasoning either way. 

Is there a reason for your preference here? I can see some value in splitting it out as a decorator, as it lets us decorate an entire test class, that needs to hit the db, but all the new tests seem to use function name as a namespacing strategy, so it wasn't so clear to me why we'd use one over the other.

Reply:
It's only a matter of personal preference - I prefer to have a decorator over adding a function argument. But frankly I didn't think much of it, and I don't think it's an impactful decision for the codebase. If you have a strong preference let me know, I can change this to a function parameter

Reply:
See below - we can punt this til later. No decisions or changes needed for right now.